### PR TITLE
Revisit faucet generosity

### PR DESCRIPTION
**Pull request by lgalabru**
Address https://github.com/blockstack/stacks-blockchain/issues/1883

### CODE REVIEW

File: src/api/routes/faucets.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ export function createFaucetRouter(db: DataStore): RouterWithAsync {
       // Only based on address for now, but we're keeping the IP in case
       // we want to escalate and implement a per IP policy
       const now = Date.now();
-      const window = 5 * 60 * 1000; // 5 minutes
+      const window = 2 * 24 * 60 * 60 * 1000; // 2 days
```
Comment (zone117x) [inline src/api/routes/faucets.ts:78]: I think people will run into issues with a 2 day throttle, but not sure of a good alternative.

Comment (psq) [reply src/api/routes/faucets.ts:78]: not too hard to "circumvent" by using multiple addresses...

**Comment by zone117x** [comment]
@aulneau @hstove @markmhx can we proceed with a 2 day faucet cooldown?

**Comment by aulneau** [comment]
I'd ask the community. It seems like two days would be too much imo.

**Comment by diwakergupta** [comment]
@lgalabru what was your rationale for changing it? If there's nothing fundamental driving the 2 day cooldown, we should just keep it low, maybe 10-15min.

**Comment by zone117x** [comment]
I think it's because the faucet can only be used around <120 times before running out, because of the large stx amount?

**Comment by zone117x** [comment]
Maybe we can have a different faucet endpoint with this aggressive throttle for those who want to try out Stacking, and preserve the behavior of the current one?

**Comment by diwakergupta** [comment]
> Maybe we can have a different faucet endpoint with this aggressive throttle for those who want to try out Stacking, and preserve the behavior of the current one?

yep, might be a good middle-ground.

> I think it's because the faucet can only be used around <120 times before running out, because of the large stx amount?

sure, but in theory we could just give the faucet a LOT more STX so this becomes a non-issue. If 10-15min is too short, maybe we can make it an hour. 2 days will definitely frustrate a LOT of people.

**Comment by zone117x** [comment]
> but in theory we could just give the faucet a LOT more STX so this becomes a non-issue

I might have misunderstood during the last meeting discussing Stacking, but I think we can't give the faucet more because it needs to give out 1/120th of the liquid supply, so if we increase the faucet supply then we have to proportionally increase the faucet disperse amount?

File: src/api/routes/faucets.ts (hunk 1 above)
Comment (lgalabru) [reply src/api/routes/faucets.ts]: Updated to 1 hour

Comment (lgalabru) [reply src/api/routes/faucets.ts]: As @psq mentioned, if for some reason (?) this backoff is an issue, users can just use a different address.

Comment (lgalabru) [reply src/api/routes/faucets.ts]: Opening https://github.com/blockstack/stacks-blockchain-api/issues/247, for the sustainable approach

**Pull request merged**